nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/_canvas.component-theme.scss

```diff
@@ -265,11 +199,11 @@
         }
 
         text.processor-type {
-            fill: utils.get-color-on-surface($color-config, $surface, 'accent');
+            fill: utils.get-color-on-surface($material-theme-color-config, $surface, 'accent');
         }
```

Comment:
do we need to have a similar treatment on the text.processor-icon? I noticed in purple mode, the icon on the canvas for a processor looks like it isn't themed and doesn't match the icon in the operation panel. The other component types seem to have the expected color

<img width="704" alt="Screenshot 2024-04-09 at 9 26 10 AM" src="https://github.com/apache/nifi/assets/713866/1a2fbc3c-a7d4-4ae6-8197-1e2719041052">

<img width="703" alt="Screenshot 2024-04-09 at 9 32 54 AM" src="https://github.com/apache/nifi/assets/713866/c0b095a9-5b5b-472e-a21c-f0e53334019a">

Reply:
This happens with the canvas components that can be colored using the paint brush icon on the operation panel. The default color is hard coded. The color after is auto calculated based on the users input.

Reply:
@james-elliott is correct. We have a hard coded 'default' color and the user can choose any color they like for this icon.